examples/docs_snippets/docs_snippets/integrations/dbt/quickstart.py

```diff
@@ -0,0 +1,100 @@
+# pyright: reportMissingImports=false
+# ruff: isort: skip_file
+
+# start_example
+from pathlib import Path
+
+from dagster import AssetExecutionContext, Definitions
+from dagster_dbt import (
+    DbtCliResource,
+    DbtProject,
+    build_schedule_from_dbt_selection,
+    dbt_assets,
+)
+
+RELATIVE_PATH_TO_MY_DBT_PROJECT = "./my_dbt_project"
+
+my_project = DbtProject(
+    project_dir=Path(__file__)
+    .joinpath("..", RELATIVE_PATH_TO_MY_DBT_PROJECT)
+    .resolve(),
+)
+
+
+@dbt_assets(manifest=my_project.manifest_path)
+def my_dbt_assets(context: AssetExecutionContext, dbt: DbtCliResource):
+    yield from dbt.cli(["build"], context=context).stream()
+
+
+my_schedule = build_schedule_from_dbt_selection(
+    [my_dbt_assets],
+    job_name="materialize_dbt_models",
+    cron_schedule="0 0 * * *",
+    dbt_select="fqn:*",
+)
+
+defs = Definitions(
+    assets=[my_dbt_assets],
+    schedules=[my_schedule],
+    resources={
+        "dbt": DbtCliResource(project_dir=my_project),
+    },
+)
+# end_example
+
+
+# start_dbt_project_example
+from pathlib import Path
+
+from dagster_dbt import DbtProject
+
+RELATIVE_PATH_TO_MY_DBT_PROJECT = "./my_dbt_project"
+
+my_project = DbtProject(
+    project_dir=Path(__file__)
+    .joinpath("..", RELATIVE_PATH_TO_MY_DBT_PROJECT)
+    .resolve(),
+)
+# end_dbt_project_example
+
+
+# Using a string to avoid ruff adding a second blank line before the dbt_assets.
+"""
```

Comment:
Can we use some `noqa` directive instead of writing these as strings?

These functions that define the snippets aren't getting called, so nothing should be executed.

Reply:
Done in [f120748](https://github.com/dagster-io/dagster/pull/21240/commits/f12074866724a77bb511bf59459ea6a67e41e172)